Cargo.toml

```diff
@@ -24,20 +24,21 @@ rustls-tls = ["hyper-rustls"]
 
 [dependencies]
 webdriver = { version = "0.46", default-features = false }
-url = "2.2.2"
-serde = { version = "1.0.103", features = ["derive"] }
-serde_json = "1.0.25"
+url = "2.2"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
I'd like to avoid these changes — the minimal listed patch versions are often there for a reason too.